pkg/aws/actuator/actuator.go

```diff
@@ -312,6 +310,8 @@ func (a *AWSActuator) sync(ctx context.Context, cr *minterv1.CredentialsRequest)
 				Message:   fmt.Sprintf("%v: %v", msg, err),
 			}
 		}
+	} else {
+		logger.Infof("unknown or missing %s annotation on admin credentials Secret, skipping reconcile", annotatorconst.AnnotationKey)
```

Comment:
Should we set a condition indicating that the CredentialsRequest object wasn't able to be reconciled b/c of this?
Specifically, I'm thinking that it is precently provisioned (status.provisioned=true), and then we hit this else clause. How is this communicated (outside of the log message)?

Reply:
Feels like a YAGNI to me, maybe if we start hitting problems with it but I find setting and clearing conditions a bit cumbersome and not necessary to do for edge cases we don't expect to have any long lasting impact. In this case we'd be waiting for secretannotator to do it's work, which should not take long and the update should requeue everything. If this were a state we could be in for more than a second or two I'd say yes, but otherwise it feels unnecessary to me.